kleva-j/iReporter
# Feature/#163386845/sign up from the frontend

#### What does this PR do?
- Ensures a user can sign-up to the app from the frontend

#### Description of the task to be completed
 - create functions to make requests to the API from the frontend to sign-up a user.

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#163386845/sign up from the frontend
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[163386845](https://www.pivotaltracker.com/story/show/163386845)
